src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.animal.AnimalComparator;
+
+
+/**
+ * Sorts all animals in the zookeep book to the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sorts the animals according to their name, ID or earliest feedtime. \n"
```

Comment:
I think feedtime should be spelt as "feed time" wherever possible (other than when the user actually types the command?) since the class is called `FeedTime`. Like in the `AnimalComparator` and `FEEDTIME_KEYWORD` as well